fedimint-client/src/module/init.rs

```diff
@@ -57,8 +58,17 @@ where
         &self.db
     }
 
+    pub fn core_api_version(&self) -> &ApiVersion {
+        &self.core_api_version
+    }
+
+    pub fn module_api_version(&self) -> &ApiVersion {
+        &self.module_api_version
+    }
+
+    // TODO: deprecate, use `module_api_version` instead
     pub fn api_version(&self) -> &ApiVersion {
-        &self.api_version
+        &self.module_api_version
```

Comment:
Why not just remove? Because backporting?

Reply:
Just wanted to make the change small. But now thinking about it - probably will help with backports indeed.